2-library/6core-logging/src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
```

Comment:
pls, create a root directory for the code,e.g. 
src/main/java/`correct path`/

Reply:
Should I transfer logback.xml to this folder? 
I thought configuration files must be in the resources directory, doesn`t it?

Reply:
configuration should be in resources you are right. 

But we still need to have a directory where we are going to write code.